test/Test/Stan/Analysis/Partial.hs

```diff
@@ -8,14 +8,31 @@ import Stan.Analysis (Analysis)
 import Stan.Inspection (Inspection (..), sortById)
 import Stan.Inspection.Partial (partialInspectionsMap)
 import Stan.NameMeta (NameMeta (..))
-import Test.Stan.Analysis.Common (itShouldStr, observationAssert, unsafeNameMeta)
+import Test.Stan.Analysis.Common (itShouldStr, noObservationAssert, observationAssert,
+                                  unsafeNameMeta)
 
 import qualified Data.Text as T
 
+import qualified Stan.Inspection.Partial as Partial
+
 
 analysisPartialSpec :: Analysis -> Spec
-analysisPartialSpec analysis = describe "Partial functions" $
+analysisPartialSpec analysis = describe "Partial functions" $ do
     forM_ (zip (sortById partialInspectionsMap) [14, 17 ..]) checkObservation
+
+    let noObservation = noObservationAssert
+            "target/Target/Partial.hs"
+            "Target.Partial"
+            analysis
+
+    it "STAN-0010: doesn't trigger on 'succ :: Natural -> Natural'" $
+        noObservation Partial.stan0010 79
+    it "STAN-0011: doesn't trigger on 'pred :: Integer -> Integer'" $
+        noObservation Partial.stan0011 82
+
+-- TODO: configure message
```

Comment:
What do you mean here?

Reply:
I mean, that I wanted to use `it` with the custom message, currently it duplicates. It requires some refactoring to achieve that. But I think it's worth doing it now instead of leaving a TODO, shouldn't be a big deal :+1:

Reply:
Oh, I see now, thanks!